configs/vite/package.json

```diff
@@ -17,12 +17,11 @@
   },
   "devDependencies": {
     "@synnaxlabs/tsconfig": "workspace:*",
-    "eslint": "^9.16.0",
+    "eslint": "^9.17.0",
     "eslint-config-synnaxlabs": "workspace:*",
-    "typescript": "^5.6.3",
+    "typescript": "^5.7.2",
     "vite": "^6.0.3",
-    "vite-plugin-dts": "^4.3.0",
-    "vite-tsconfig-paths": "^5.1.4"
```

Comment:
Why did you remove ts config paths? Don't we need it?

Reply:
It's only needed in the console package (where it still exists), it is not used in the main vite plugin.